war1tool.cpp

```diff
@@ -1911,23 +1911,23 @@ void MuxAllIntroVideos() {
 	fprintf(mylist, "file '%s/%s/%s'\n", Dir, SOUND_PATH, "intro_4.wav");
 	#endif
 	fclose(mylist);
-	const char* cmd3a = "ffmpeg -y -f concat -safe 0 -i %s "
-		"%s/%s/cave_a.wav";
+	const char* cmd3a = "ffmpeg -y -f concat -safe 0 -i \"%s\" "
+		"\"%s/%s/cave_a.wav\"";
 	snprintf(cmd, STATIC_CMD_SIZE - 1, cmd3a, listfileSound, Dir, SOUND_PATH);
 	system(cmd);
 
-	const char* cmd3 = "ffmpeg -y -i %s/%s/cave_v.avi -i %s/%s/cave_a.wav "
-		"-codec:v libtheora -qscale:v 31 -pix_fmt yuv420p -vb 4000k -codec:a libvorbis -qscale:a 15 %s/%s/cave.ogv";
+	const char* cmd3 = "ffmpeg -y -i \"%s/%s/cave_v.avi\" -i \"%s/%s/cave_a.wav\" "
+		"-codec:v libtheora -qscale:v 31 -pix_fmt yuv420p -vb 4000k -codec:a libvorbis -qscale:a 15 \"%s/%s/cave.ogv\" ";
 	snprintf(cmd, STATIC_CMD_SIZE - 1, cmd3, Dir, VIDEO_PATH, Dir, SOUND_PATH, Dir, VIDEO_PATH);
 	system(cmd);
 
 	// Title sequence
 	#ifdef WIN32
-	const char* cmd4t = "ffmpeg -i %s/%s/title.avi -codec:v huffyuv "
-	    "%s/%s/title_s.avi"; // video filter crop not supported on windows
+	const char* cmd4t = "ffmpeg -i \"%s/%s/title.avi\" -codec:v huffyuv "
+	    "\"%s/%s/title_s.avi\""; // video filter crop not supported on windows
 	#else
-	const char* cmd4t = "ffmpeg -i %s/%s/title.avi -codec:v huffyuv "
-		"-vf 'crop=640:288:0:0' %s/%s/title_s.avi";
+	const char* cmd4t = "ffmpeg -i \"%s/%s/title.avi\" -codec:v huffyuv "
+		"-vf 'crop=640:288:0:0'\" %s/%s/title_s.avi\" ";
```

Comment:
```suggestion
		"-vf 'crop=640:288:0:0' \"%s/%s/title_s.avi\" ";
```
Misplaced space.
Unsure why you add extra space at the end...

I created https://github.com/Wargus/war1gus/tree/quotes_fix
which is mostly a clean (from git history) work of your branch.

Reply:
Nice catch, thanks.